app/models/url_rewrite.rb

```diff
@@ -16,6 +16,8 @@ class UrlRewrite < ApplicationRecord
   validates :from, :to, presence: true
   validates :from, format: { with: %r{\A/.*\z}, message: "should start with /" } # rubocop:disable Rails/I18nLocaleTexts
 
+  before_validation :strip_whitespace
```

Comment:
What is the intended use case here? Doesn't the existing validation already address this?